examples/widgets/text_cursor.py

```diff
@@ -0,0 +1,313 @@
+"""
+==========
+Textcursor
+==========
+
+Display a data cursor including a text box, which shows the plot point close
+to the mouse pointer.
+
+The new cursor inherits from `~matplotlib.widgets.Cursor` and demonstrates the
+creation of new widgets and their event callbacks.
+
+.. note::
+    The figure related to this example does not show the cursor, because that
+    figure is automatically created in a build queue, where the first mouse
+    movement, which triggers the cursor creation, is missing.
```

Comment:
You could send a first fake event. But not sure if it's worth  it.

Reply:
See the commit e989183725dc3e0091ea8ea74a6701ea730b32ea where I removed the following code

```python
# In the gallery picture, we cannot see the cursor.
# The cursor is displayed after the first mouse move event.
# There is no mouse move when using automatic scripts forgenerating the
# documentation. The onmove event needs to occur after the plt.show() command,
# because this one creates a fresh and clean canvas. Without the cursor or
# its text. And after the plt.show(), we cannot execute more instructions.
# The following code draws the figure (without showing it) and calls the mouse
# event manually. The resulting figure is saved to a file. But it cannot be
# shown, because this first clears the canvas and therefore removes the cursor.
from matplotlib.backend_bases import MouseEvent
location = np.array([0, 10])
location = ax.transData.transform(location)
event = MouseEvent(name='motion_notify_event', button=None, key=None,
   x=location[0], y=location[1], canvas=ax.figure.canvas)
ax.figure.canvas.draw()
cursor.onmove(event)
fig.savefig('text_cursor.png')
```

and added the note. Sending the fake event AND having plt.show also does not work. So I removed all that code and did not tried to add e.g. a "draw" event callback, because that might confuse the reader and one might have trouble distinguishing between stuff one needs for the core example and stuff needed for the sphinx gallery.

So I came to the conclusion: It's not worth it.